src/lib/core/overlay/overlay-directives.ts

```diff
@@ -76,30 +75,10 @@ export class ConnectedOverlayDirective implements OnDestroy {
   @Input() positions: ConnectionPositionPair[];
 
   /** The offset in pixels for the overlay connection point on the x-axis */
-  @Input()
-  get offsetX(): number {
-    return this._offsetX;
-  }
-
-  set offsetX(offsetX: number) {
-    this._offsetX = offsetX;
-    if (this._position) {
-      this._position.withOffsetX(offsetX);
-    }
-  }
+  @Input() offsetX: number = 0;
```

Comment:
A getter/setter combination transpiles to this in ES5:

```
Object.defineProperty(A.prototype, "thing", {
    get: function () {
        return this._thing;
    },
    set: function (value) {
        this._thing = value;
    },
    enumerable: true,
    configurable: true
});
```

I think that it can definitely start adding up over an entire project.